packages/mdc-list/mdc-list.scss

```diff
@@ -75,6 +76,11 @@ $mdc-list-side-padding: 16px;
   justify-content: flex-start;
   height: 48px;
 
+  &--selected {
```

Comment:
I believe the `mdc-list-item-graphic-fill-color` and `mdc-list-item-graphic-ink-color` mixins should cover that use case, unless I'm misunderstanding your question.

Reply:
I meant a way to do it by emitting fewer styles rather than extra styles that are kind of redundant overrides. But if we think it's more common to color both then we should probably go ahead and do that.